sdk/Cargo.toml

```diff
@@ -8,11 +8,13 @@ repository = "https://github.com/oxidecomputer/oxide.rs"
 readme = "README.md"
 
 [dependencies]
+anyhow = { workspace = true }
 base64 = { workspace = true }
 chrono = { workspace = true }
 clap = { workspace = true, optional = true }
 dirs = { workspace = true }
 futures = { workspace = true }
+indicatif = { workspace = true }
```

Comment:
This looks to be unused now.

Reply:
Whoops, fixed in 7cc0419cc159d5b4a48a8d8c9bec1dddc7a53cb3